package.json

```diff
@@ -14,10 +14,16 @@
     "lint": "echo 'Linting all workspaces...' && yarn constraints && yarn workspaces foreach -p run lint",
     "build:dev": "yarn workspace @segment/analytics-next build:dev",
     "dev": "yarn workspace @segment/analytics-next exec make dev",
-    "postinstall": "husky install"
+    "postinstall": "husky install",
+    "version": "changeset version && yarn version-run-all",
+    "release": "yarn prepublish-run-all && changeset publish",
+    "prepublish-run-all": "yarn workspaces foreach -pt --no-private run prepublish",
```

Comment:
`changeset` does not (at first glance) to support lifecycle scripts natively. I created an issue.